### ISSUE TITLE
cannot get samba root file path

### ISSUE THREAD

**Issue by Tangwego** [hierynomus/smbj#768]
cannot get samba root file path
like \\192.168.1.10\ 
only support \\192.168.1.10\dir\...  now?

**Comment by hierynomus** [comment]
Yes, smbj currently does not natively support the smb-rpc call needed to list all shares of a host.

**Comment by Tangwego** [comment]
I need visit the root directory, but it seems don't support.
